dist/placeholder-buster.txt

```diff
@@ -129,6 +129,9 @@ rpmseek.com###lbanner_top
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/41#issuecomment-515740473
 edition.cnn.com###story-bottom
 
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/44
+manuall.*,handleidingkwijt.com,shuomingshu88.com##.col-sm-9 > div:has-text(advertisement)
```

Comment:
Can you provide test links for the other domains as well?

Reply:
They're pretty much all the same site, just that two of the regional versions had to use different domain names. But nevertheless I've done that now.